authorizerd.go

```diff
@@ -89,7 +100,7 @@ func New(opts ...Option) (Providerd, error) {
 		return nil, errors.Wrap(err, "error create pubkeyd")
 	}
 
-	if prov.policyd, err = policy.NewPolicyd(
+	if prov.policyd, err = policy.New(
```

Comment:
```suggestion
	if prov.policyd, err = policyd.New(
```

Reply:
you can do that on another PR

Reply:
OK, I will do it after this PR is merged.
Or this one is better?
```suggestion
	if prov.policyd, err = policy.NewDaemon(
```

Reply:
I don't think so, New means Instantiate policyd instance, policyd is daemon but this function is just construct instance

Reply:
For me, I think policy.New() means creating a policy object instead of the policy daemon....
But if we rename the package name to policyd then it make no sense for the policy object...
So I think rename the function name should be better.
What do u think?

Reply:
Yes we just create policy object to start policyd

Reply:
I think policyd.New().Start... is better